Part2_Mapplots/ex2_0_intro_scatter.ipynb

```diff
@@ -480,7 +476,7 @@
    "source": [
     "#### Note\n",
     "\n",
-    " > The colorbar is too big for the map! However, this is not straightforward to correct... In [Exercise 2.4](ex2_4_colorbars.ipynb) we will learn how to fix this."
+    " > The colorbar is too big for the map (also because the aspect ratio of the map is fixed)! However, this is not straightforward to correct... In [Exercise 2.4](ex2_4_colorbars.ipynb) we will learn how to fix this."
```

Comment:
```suggestion
    " > The colorbar is too big for the map (also because the aspect ratio of the map is fixed). However, this is not straightforward to correct... In [Exercise 2.4](ex2_4_colorbars.ipynb) we will learn how to fix this."
```